tests/cases/fourslash/overloadOnConstCallSignature.ts

```diff
@@ -11,8 +11,8 @@
 
 goTo.marker('1');
 verify.signatureHelpCountIs(4);
-verify.currentSignatureHelpIs('foo(name: string): string');
+verify.currentSignatureHelpIs('foo(name: \'order\'): string');

```

Comment:
This does seem weird, but this might be a problem with the strategy for signature help. Ideally, you would see the string overload because it is the most general.